doc/source/index.rst

```diff
@@ -0,0 +1,48 @@
+.. qutip_jax
+
+.. figure:: figures/logo.png
+    :align: center
+    :width: 5in
+
+*********************************************************
+qutip-jax |version|: Jax-backend for QuTiP 
```

Comment:
Is this part being rendered properly?  The duplication of the title looks a bit odd.
![image](https://github.com/qutip/qutip-jax/assets/57414022/f65f2d18-7d48-4d17-a6f9-208a7540866d)

Reply:
Removed the duplicate.